### PR TITLE
Update lesson format

### LINKED ISSUE #22: Update and add more exercises to format.md episode
Add two types of exercises in this episode:
- ask the learner to search for a common format (mentioned in this episode) in https://fairsharing.org/ and inspect the result.
- ask the learner to have a look at the checklist https://zenodo.org/record/3405141#.X4AeiHUzZhF and find those that are related to the file format.

Consider CF convention for CMIP data that is related to NetCDF format as an example. For other climate-related domains, add any conventions/standards that are related to a specific format.
Some resources: 
https://gmd.copernicus.org/articles/11/3659/2018/gmd-11-3659-2018.pdf
https://cfconventions.org/index.html
https://pcmdi.llnl.gov/CMIP6/Guide/dataUsers.html

Also, polish the exercises and lessons to illustrate clearly about common/ acceptable/ recommended formats.

### PR DESCRIPTION
**Pull request by SarahAlidoost**
Closes #22 #23

### CODE REVIEW

File: _episodes/format.md [hunk 1]
```diff
@@ -57,190 +60,215 @@ Tabular data have specific properties that are better supported by the spreadshe
...
+>> ## Solution
+>>
+>> - I1:(meta) data use a formal, accessible, shared, and broadly applicable
+>>   language for knowledge representation. For example, if data is provided in
+>>   commonly understood and preferably open formats.
+>>
+>> - R1.3: (meta)data meet domain-relevant community standards e.g. those for
+>   data formats.
+> {: .solution}
+{: .challenge}
```
Comment (Peter9192) [inline _episodes/format.md]: Maybe we can make this consistent with e.g. episode on data access. There it's a checkbox like: "This is what go-FAIR.org has to say about Accessible"

File: _episodes/format.md [hunk 2]
```diff
@@ -57,190 +60,215 @@ Tabular data have specific properties that are better supported by the spreadshe
...
+and paper, textual, video/audio, and image/figure. Some of them are more in line
+with the FAIR principles than some others. Here are some recommended formats for
+different types:
+
+<table style="width:40%">
+  <thead>
+  <tr>
+      <th>Data type</th>
+      <th>Data format</th>
+    </tr>
```
Comment (Peter9192) [inline _episodes/format.md:84]: Wouldn't markdown syntax be simpler for this table? https://idratherbewriting.com/documentation-theme-jekyll/mydoc_tables.html#multimarkdown-tables

Or does that not work?

File: _episodes/format.md [hunk 3]
```diff
@@ -57,190 +60,215 @@ Tabular data have specific properties that are better supported by the spreadshe
...
+  are uploaded in certain file formats. Therefore, you may have to keep some
+  data files in multiple formats.
+
+![format_support_matrix]({{ page.root }}/fig/format_support_matrix.png) If we
+store the data in a more open or widely supported format, it will have the
+greatest re-usability in the future.
 
 > ## Non/proprietary format
 >
 > File formats may be either proprietary or non-proprietary(open or free):
```
Comment (Peter9192) [inline _episodes/format.md]:
```suggestion
> File formats may be either proprietary or non-proprietary (open or free):
```

**Review by Peter9192 — APPROVED** [review]
Looks good! A few small comments, but feel free to merge it when you're ready :-)

File: _episodes/format.md (hunk 1 above)
Comment (SarahAlidoost) [reply _episodes/format.md]: Thanks, Done!

File: _episodes/format.md (hunk 2 above)
Comment (SarahAlidoost) [reply _episodes/format.md:84]: I couldn't change the width of the table using markdown syntax.

**Comment by SarahAlidoost** [comment]
> Looks good! A few small comments, but feel free to merge it when you're ready :-)

Thank you for reviewing. If you could resolve the conversations, I will merge it.

**Comment by Peter9192** [comment]
Thanks Sarah, sorry for the delay.

**Pull request merged**